src/main/kotlin/com/chattriggers/ctjs/engine/module/ModuleUpdater.kt

```diff
@@ -98,18 +97,36 @@ object ModuleUpdater {
         }
     }
 
-    fun importModule(moduleName: String): List<Module> {
-        if (cachedModules.any { it.name == moduleName }) return emptyList()
+    fun importModule(moduleName: String, requiredBy: String? = null): List<Module> {
+        if (cachedModules.any {
+                return@any if (it.name == moduleName) {
```

Comment:
Unnecessary `return@any`, also could we pull this into a variable? I'm not a fan of huge if statement conditions like this